Source/Hamakaze/KDU.vcxproj

```diff
@@ -32,6 +33,7 @@
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <CharacterSet>Unicode</CharacterSet>
     <SpectreMitigation>false</SpectreMitigation>
+    <UseOfMfc>Static</UseOfMfc>
```

Comment:
What is the benefits of this? This project does not use any kind of MFC, nor they usage are in plans.